plugins/woocommerce/tests/e2e-pw/bin/test-env-setup.sh

```diff
@@ -12,6 +12,9 @@ wp-env run tests-cli wp rewrite structure '/%postname%/' --hard
 echo -e 'Activate Filter Setter utility plugin \n'
 wp-env run tests-cli wp plugin activate filter-setter
 
+echo -e 'Activate Process Waiting Actions utility plugin \n'
+wp-env run tests-cli wp plugin activate process-waiting-actions
```

Comment:
My understanding is that data gets processed for the analytics page asynchronously.  In our tests, we add a bunch of orders all at once, and there's no guarantee of the analytics being updated in X amount of time.  Barry was kind enough to put together this plugin that gives us a network request we can make to start processing that data immediately.  All this plugin does is that it immediately starts processing the jobs in the Action Scheduler queue.

I don't know if we'll always need this, or if someday Analytics will be re-worked to show realtime numbers. But for now, it's the only way we can run any tests on Analytics in a reasonable amount of time.

Reply:
Thank you so much for the explanation @lanej0!

It would be great if could add a comment to the bash script explaining this 🙏 This will help sharing knowledge with other folks contributing to the project.